tv_apps/smart-system/fxa/js/fxam_key_navigation.js

```diff
@@ -0,0 +1,108 @@
+/* global KeyNavigationAdapter */
+/* global SpatialNavigator */
+/* exported FxaModuleKeyNavigation */
+
+'use strict';
+
+(function (exports) {
+
+  function getElements(elementNames) {
+    var elements = [];
+
+    elementNames.forEach(selector => {
+      var element = document.querySelector(selector);
+      if (element) {
+        elements.push(element);
+      }
+    });
+
+    return elements;
+  }
+
+  var FxaModuleKeyNavigation = {
+
+    enabled: false,
+
+    spatialNavigator: null,
+
+    keyNavigationAdapter: null,
+
+    init(elementNames) {
```

Comment:
Yes, trace back to fxam_manager.js, the init function only runs once on window ```load``` event.